### PR TITLE
refactor(bluebird): remove Bluebird.bind

### PR DESCRIPTION
**Pull request by maximelkin**

### CODE REVIEW

File: lib/migrate/Migrator.js [hunk 1]
```diff
@@ -475,7 +474,7 @@ class Migrator {
   _waterfallBatch(batchNo, migrations, direction, trx) {
     const trxOrKnex = trx || this.knex;
     const { tableName, schemaName, disableTransactions } = this.config;
-    let current = Bluebird.bind({ failed: false, failedOn: 0 });
+    let current = Promise.resolve();
```
Comment (kibertoad) [inline lib/migrate/Migrator.js]: are those fields not used anywhere?

Comment (maximelkin) [reply lib/migrate/Migrator.js]: I didn't found any usages in project

Comment (maximelkin) [reply lib/migrate/Migrator.js]: 7a6937fdb329d0d0b29cd51fc28aea06e6903e0e first revision where failedOn exists, i think it just partially implemented feature

Comment (kibertoad) [reply lib/migrate/Migrator.js]: Fair enough, guess we can drop it.

**Comment by kibertoad** [comment]
Sorry, there is a merge conflict now after refactoring of a seeder. Could you resolve them?

**Comment by maximelkin** [comment]
Ok, i will resolve it later

File: lib/seed/Seeder.js [hunk 2]
```diff
@@ -28,7 +20,7 @@ class Seeder {
   // Runs seed files for the given environment.
   async run(config) {
     this.config = this.setConfig(config);
-    const [all] = await this._seedData();
+    const all = await this._listAll();
```
Comment (kibertoad) [inline lib/seed/Seeder.js]: Will types changed in https://github.com/knex/knex/pull/3438/files still be correct after this change?

File: lib/seed/Seeder.js [hunk 3]
```diff
@@ -103,22 +85,14 @@ class Seeder {
     );
   }
 
-  // Generates the stub template for the current seed file, returning a compiled template.
```
Comment (kibertoad) [inline lib/seed/Seeder.js]: @lorefnon I assume we stopped using it after your refactoring?

**Comment by kibertoad** [comment]
@maximelkin Merged your other PR, that seems to have introduced a minor conflict as well.

File: lib/seed/Seeder.js (hunk 3 above)
Comment (lorefnon) [reply lib/seed/Seeder.js]: Yeah, i moved the lodash template usage to a shared utility which we can potentially use from other places as well.

File: lib/seed/Seeder.js (hunk 2 above)
Comment (maximelkin) [reply lib/seed/Seeder.js]: _seedData is just _listAll, but wrapping result inside promise into []

**Comment by maximelkin** [comment]
@kibertoad done

**Pull request merged**